src/test/java/seedu/application/logic/commands/RemindCommandStub.java

```diff
@@ -0,0 +1,38 @@
+package seedu.application.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.application.model.Model;
+import seedu.application.model.application.UpcomingInterviewPredicateStub;
+
+/**
+ * Reminds users of upcoming interviews within 1 week from now stored in CinternS.
+ * The predicate used in this stub has a pre-configured date and time reference
+ * and does not follow the system clock.
+ */
+public class RemindCommandStub extends Command {
```

Comment:
And `RemindCommandStub` can extend `RemindCommand` too? So that a few repeated code can be avoided.